SwiftCompilerSources/Sources/Optimizer/FunctionPasses/CopyToBorrowOptimization.swift

```diff
@@ -360,6 +360,14 @@ private extension Value {
   }
 
   var lookThroughForwardingInstructions: Value {
+    if let bfi = definingInstruction as? BorrowedFromInst,
+       let forwardedOp = bfi.singleForwardedOperand,
+       let phi = Phi(forwardedOp.value), !phi.isReborrow,
```

Comment:
you can just use `bfi.borrowedPhi` instead of getting it through the singleForwardedOperand